integration_tests/tests/consistency/consistency__audit_table.sql

```diff
@@ -0,0 +1,67 @@
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with prod as (
+    select
+        connector_id,
+        table_name,
+        count(*) as total_records
+    from {{ target.schema }}_fivetran_platform_prod.fivetran_platform__audit_table
+    group by 1, 2
+),
+
+dev as (
+    select
+        connector_id,
+        table_name,
+        count(*) as total_records
+    from {{ target.schema }}_fivetran_platform_dev.fivetran_platform__audit_table
+    group by 1, 2
+),
+
+final_consistency_check as (
+    select 
+        prod.connector_id,
+        prod.table_name,
+        prod.total_records as prod_total,
+        dev.total_records as dev_total
+    from prod
+    left join dev 
+        on dev.connector_id = prod.connector_id
+        and dev.table_name = prod.table_name
+),
+
+-- Checking to ensure the dev totals match the prod totals
+consistency_check as (
+    select *
+    from final_consistency_check
+    where prod_total != dev_total
+),
+
+-- The current release changes the row count of the audit table model intentionally.
+-- The below queries prove the records that do not match are still accurate by checking the source.
```

Comment:
@fivetran-joemarkiewicz Quick question--are these tests in anticipation of the model changes from [this PR](https://github.com/fivetran/dbt_fivetran_log/pull/116) being merged?

Reply:
@fivetran-joemarkiewicz Got it thanks! Maybe changing the comment to something like "For use when the current release..." would solve any confusion.

Reply:
I like that suggestion! Just made the update.